util/ModeratorAction.cpp

```diff
@@ -157,8 +157,7 @@ namespace {
     std::string GenerateSystemName() {
         static std::vector<std::string> star_names = UserStringList("STAR_NAMES");
 
-        const ObjectMap& objects = Objects();
-        std::vector<std::shared_ptr<const System>> systems = objects.all<System>();
+        auto systems = Objects().all<System>();
```

Comment:
Can probably iterate over systems directly with `all<System>` below without this

Reply:
See d9fb4f9bf58fb845bf4547dd4b9e1f53bc7df8e0